packages/ember-application/lib/system/application.js

```diff
@@ -980,6 +988,71 @@ Application.reopenClass({
     @param initializer {Object}
     @public
    */
+
+   /**
```

Comment:
Looks like this is causing the style checking tests to fail, I *think* it should be one space out-dented, but you can double check by running the tests locally.